Gemfile

```diff
@@ -55,6 +55,7 @@ gem "manageiq-ssh-util",                "~>0.1.1",           :require => false
 gem "memoist",                          "~>0.16.0",          :require => false
 gem "money",                            "~>6.13.5",          :require => false
 gem "more_core_extensions"                                                     # min version should be set in manageiq-gems-pending, not here
+gem "net-ftp",                          "~>0.1.2",           :require => false
```

Comment:
Not sure if this should belong here or manageiq-gems-pending

Reply:
yeah, it also blew up in gems pending in the same require so they both will need the same solution... note, it was blowing up here so will need to either move the calling code out of core or let gems pending drive the gem specification